sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/IncrementalRegressionTests.java

```diff
@@ -26,6 +26,170 @@ public DBSPCompiler testCompiler() {
         return new DBSPCompiler(options);
     }
 
+    @Test
+    public void issue3164() {
+        this.getCCS("""
+               CREATE FUNCTION parse_ts(input VARCHAR) RETURNS TIMESTAMP NOT NULL;
```

Comment:
maybe the test could be simplified. You probably don't need all this for a minimal repro.